Rubberduck.CodeAnalysis/Inspections/Concrete/UnreachableCaseEvaluation/UnreachableCaseInspector.cs

```diff
@@ -98,23 +98,23 @@ internal class UnreachableCaseInspector : IUnreachableCaseInspector
 
         private UnreachableCaseInspection.CaseInspectionResultType? InvalidRangeExpressionsType(ICollection<IRangeClauseExpression> rangeClauseExpressions)
         {
-            if (rangeClauseExpressions.Any(expr => expr.IsMismatch))
+            if (rangeClauseExpressions.Any(expr => expr?.IsMismatch == true))
```

Comment:
All the conditions have been changed, making the original comment outdated.
However, it still remains that the numerous `expr?` should be converted to one guard clause at the start.

Reply:
How though? We're inside an `Any` method...